docs/pages/database-access/reference/aws.mdx

```diff
@@ -154,6 +154,14 @@ policies for each discovery type are shown below.
   </TabItem>
 </Tabs>
 
+<Admonition type="note">
+Teleport uses `rds:ModifyDBInstance` and `rds:DescribeDBClusters` to
+automatically enable [IAM
+authentication](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.html)
+on the RDS instance and the Aurora cluster respectively. You can omit these
```

Comment:
```suggestion
on the RDS instance and the Aurora cluster, respectively. You can omit these
```
[Reference](https://www.internationalscienceediting.com/how-to-use-respectively-correctly/#:~:text=If%20at%20the%20end%20of,at%20a%20beamwidth%20of%20Z.)